src/Config.UserSecrets/build/netstandard2.0/Microsoft.Extensions.Configuration.UserSecrets.props

```diff
@@ -0,0 +1,8 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <!-- This capability is only available if UserSecretsId is set -->
+    <ProjectCapabilities Include="LocalUserSecrets" Condition=" '$(UserSecretsId)' != '' " />
```

Comment:
Per our email conversation, a capability is generally supposed to reflect what a project is *capable of*, rather than a reflection of its current settings. If `UserSecretsId` can be set while the project is open, then the project capability would generally be expected to be defined either way. I don't know how much it matters in this particular case, but I expect we should only deviate from that norm if you have a particular reason to do so. If so, what it is?

Reply:
I guess I misinterpreted what you were suggesting in our email conversation. If unconditional capabilities is the normal route, lets do that instead. We can handle projects that are missing UserSecretsId from the VS instead.

Updated the PR to drop this condition.